forch/faucet_event_client.py

```diff
@@ -212,9 +212,11 @@ def _dispatch_faucet_event(self, target, target_event):
     def _should_log_event(self, event):
         return event and os.getenv('FAUCET_EVENT_DEBUG')
 
-    def next_event(self, blocking=False):
+    def next_event(self, get_config_hash_clashed, blocking=False):
```

Comment:
this is not right -- the concept of a config hash shouldn't bleed down into the event client itself

Reply:
removed